app/features/dynamo/prompt/dynamo-prompt.txt

```diff
@@ -12,6 +12,6 @@ Formatting:
 -----------------------------
 {format_instructions}
 
-Respond only according to the format instructions. The examples included are best responses noted by an input and output example.
+Respond only according to the format instructions. The examples included are best responses noted by an input and output example. You must provide the response in "{lang}" language. 
```

Comment:
Multilingual support isn't in the requirement specification, so you can remove it, as LLMs aren't that good at machine translations anyway.

Reply:
Multilingual support is a competitive advantage for our services, as we'll increase our approach and reach customers worldwide. This was explained in a previous Weekly Team Retro meeting. About Machine Traslation, these articles argue that Gemini-1.5 is really good at Multilingual Mastery: https://medium.com/@cognidownunder/google-gemini-1-5-pro-revolutionizing-ai-with-expanded-context-and-multilingual-mastery-3c80a8cb6ace https://bito.ai/blog/gemini-1-5-pro-vs-gpt-4-turbo-benchmarks/#:~:text=In%20its%20analysis%20of%20vast,large%20as%2010%20million%20tokens. and this scientific papers states the same approach: https://arxiv.org/html/2403.05530v2  as it showed that Gemini 1.5 has 'continued improvement in next-token prediction and near-perfect retrieval (>99%) ' for multilingual purposes.